src/pages/home/components/Sidebar/Sidebar.module.scss

```
@@ -4,17 +4,49 @@
 .container {
   width: 100%;
   height: 100%;
+  height: 100%;
+  max-height: 80vh;
+
+  flex-direction: column;
 }
 
 .accordionContainer {
-  width: 100%;
-
   display: flex;
   flex-direction: column;
   flex-grow: 1;
   gap: 0.8rem;
 }
 
+.scrollableContent {
+  width: 100%;
+  overflow-y: auto;
+
+  flex: 1;
+
+  padding: 0 5px;
```

Utilize a unidade rem para padding e margin e para elementos relativos

Nesse caso 0.5rem = 5px

```suggestion
  padding: 0 0.5rem;
```

caso queira saber um pouco melhor sobre quais unidades utilizar e onde: https://www.linkedin.com/pulse/unidades-css-um-guia-completo-alexandre-gomes/